File: swagger/models/get_locations_request.py
from .base_model import BaseModel


class GetLocationsRequest(BaseModel):
	"""
	no description available

	Attributes:

		limit
		Number of results to include, defaults to 100

		offset
		Page offset, defaults to 0.
	"""

	input_map = {
		'Limit': 'limit',
		'Offset': 'offset',
		}

	output_map = {
		'limit': 'Limit',
		'offset': 'Offset',
		}

File: swagger/Models/GetLocationsRequest.php
<?php

namespace App\MindBody\Models;

/*
 * no description available
 */
class GetLocationsRequest extends BaseModel
{
	/*
	 * Number of results to include, defaults to 100
	 * 
	 * @var integer
	 */
	public $limit;
	/*
	 * Page offset, defaults to 0.
	 * 
	 * @var integer
	 */
	public $offset;
}
